src/libecalc/presentation/exporter/queries.py

```diff
@@ -134,11 +135,13 @@ def __init__(
         consumer_categories: Optional[List[str]] = None,
         fuel_type_category: Optional[str] = None,
         emission_type: Optional[str] = None,
+        volume_output: Optional[bool] = False,  # Relevant for venting emitters of category storage or loading only
```

Comment:
Can we use a separate VolumeQuery instead? This seems like something that could end up being separate from emissions in the future